src/rgw/rgw_op.cc

```diff
@@ -6318,7 +6327,12 @@ void RGWCompleteMultipart::execute(optional_yield y)
 		     << " ret=" << op_ret << dendl;
     return;
   }
+
+  jspan_context trace_ctx(false, false);
+  extract_span_context(meta_obj->get_attrs(), trace_ctx);
+  multipart_trace = tracing::rgw::tracer.add_span(name(), trace_ctx);
```

Comment:
need to make sure that the `name()` is different than the one from the generic op

Reply:
initial trace name changed to be "multipart_upload "
then once we get the upload_id, we update the name to be "multipart_upload <upload_id>"

Reply:
so the spans that come from PutObj op will be named "put_obj", but they will be presented under the multipart_upload trace. so it won't be confusing.